docs/developing/publishing-apps.md

```diff
@@ -0,0 +1,105 @@
+
+You can select `openSource` and specify an OSI-approved license, or select `proprietary` and embed the full text of the license. If you choose a proprietary license, which may contain more restrictive permissions, Sandstorm will display the license to the user and have them accept it before they are able to use the app. If your app does not contain more restrictive permissions, you may consider releasing under `none` to avoid this. Finally, `publicDomain` is also available, but it is preferable to use a permissive open source license instead.
+
+You may also need to include `notices` if your app is required to display any third-party copyright notices, for example due to use of third-party open source libraries.
+
+#### categories
+
+You may select the market categories to which your app belongs. You may select multiple, but you may be asked to make changes if the market moderators feel they are inappropriate.
+
+Currently, the following categories are accepted: productivity, communications, social, webPublishing, office, developerTools, science, graphics, media, games, and other.
+
+#### author
+
+The author can be an individual, organization, or even a pseudo-identity representing the app. In order for users to be able to verify the author of a package, the app author can only be identified by PGP key. The Sandstorm team recommends using keybase.io.
+
+* The `upstreamAuthor` is the name of the primary author of the original app. This indicates the author identified by the key ported the app, which was developed by someone else. If the original author is the one publishing the app, do not include this.
+* The `contactEmail` is the address to contact for any issues with this app. This both includes administrative issues with the app market listing as well as end user support requests. It is very important that this email be monitored.
+* The `pgpSignature` field is where you embed a signed ASCII statement verifying that you are the author of this app package.
+
+#### pgpKeyring
+
+This is where you embed a keyring in GPG keyring format containing the public key needed to verify your signature.
+
+#### description
+
+You should embed a description of your app in Github-flavored Markdown. It may not contain HTML or image tags, as you can attach screenshots separately.
+
+#### shortDescription
+
+Include two or three words here that briefly characterize your app.
+
+#### screenshots
+
+You can attach a number of screenshots here. You should specify the height and width of the picture here in "device-independent pixels". You may embed PNGs or JPGs here.
+
+#### changeLog
+
+Here you may embed a log of changes in Github-flavored Markdown. It is recommended to format this with a H1 heading for each release followed by a bullet list of changes.
```

Comment:
Optional: Link to the Sandstorm changelog as an example, or some app like Etherpad, e.g. https://raw.githubusercontent.com/kentonv/etherpad-lite/sandstorm/CHANGELOG.md